unlock-app/src/components/content/events-collection/EventCollectionSettings.tsx

```diff
@@ -0,0 +1,152 @@
+'use client'
+import { BsArrowLeft as ArrowBackIcon } from 'react-icons/bs'
+import { SettingsContext } from '~/components/interface/locks/Settings'
+import { Tab } from '@headlessui/react'
+
+import { ReactNode, useState } from 'react'
+
+import Link from 'next/link'
+import { EventCollection } from '@unlock-protocol/unlock-js'
+import { General } from './settings/General'
+import { useAuth } from '~/contexts/AuthenticationContext'
+
+interface EventCollectionSettingsProps {
+  eventCollection: EventCollection
+}
+
+export const EventCollectionSettings = ({
+  eventCollection,
+}: EventCollectionSettingsProps) => {
+  const [selectedIndex, setSelectedIndex] = useState(0)
+  const { account } = useAuth()
+
+  const isManager =
```

Comment:
We probably want to move that to dinner utile file some we're doing it multiple times

Reply:
sure thing!